PrebidMobile/PrebidMobile-core/src/main/java/org/prebid/mobile/Util.java

```diff
@@ -560,4 +559,23 @@ private static void removeUsedCustomTargetingForDFP(Object publisherAdRequest) {
         return result;
     }
 
+    public static String generateInstreamUriForGam(String adUnit, int w, int h, Map<String, String> keywords) {
+        String uri = "https://pubads.g.doubleclick.net/gampad/ads?";
+        if (TextUtils.isEmpty(adUnit)) {
+            throw new IllegalArgumentException("adUnit should not be empty");
+        }
+        if (w <= 0 || h <= 0) {
+            throw new IllegalArgumentException("w or h should be positive numbers");
+        }
+        uri = uri + "sz=" + w + "x" + h + "&iu=" + adUnit + "&impl=s&gdfp_req=1&env=vp&output=vast&unviewed_position_start=1";
```

Comment:
I have a general question regarding a required parameter: `output`

1. Does it have a correlation with `OpenRTB.Protocols`
2. [Google Docs](https://support.google.com/admanager/table/9749596?hl=en&ref_topic=2480647&visit_id=637401804158968284-34071955&rd=2) says 

> if your video player uses the IMA SDK, the output parameter for a video ad request will always be set to output=xml_vast4

And it is true if take a look at a  real request
<img width="905" alt="Screenshot 2020-11-05 at 15 54 46" src="https://user-images.githubusercontent.com/8141931/98249722-41ca0b80-1f7f-11eb-9689-a8ce8cac74e4.png">

Should we handle it ?

Reply:
Fixed.